lib/common/__init__.py

```diff
@@ -214,54 +215,18 @@ def parse_proofmode_data(proofmode_path):
             if date_create is None or current_date_create < date_create:
                 date_create = current_date_create
 
-            if os.path.splitext(file)[1] == ".csv" and "batchproof.csv" not in file:
+            if os.path.splitext(file)[1] == ".json" and "batchproof" not in file:
 
                 base_file_name = os.path.splitext(file)[0]
                 base_file_name = os.path.splitext(base_file_name)[0]
 
-                data = proofmode.read(file).decode("utf-8")
-
-                pgp = proofmode.read(base_file_name + ".asc").decode("utf-8")
-
-                heading = None
-
-                # Convert CSV to JSON
-                csv_reader = csv.reader(data.splitlines(), delimiter=",")
-                json_metadata_template = {}
-                json_metadata = {"proofs": []}
-                for row in csv_reader:
-                    json_metadata_row = {}
-                    # Read Heading
-                    if heading == None:
-                        column_index = 0
-                        for col_name in row:
-                            json_metadata_template[col_name] = ""
-                            column_index += 1
-                        # Add dumy Lat/Long in case proof mode does not generate them
-                        json_metadata_template["Location.Latitude"] = 0
-                        json_metadata_template["Location.Longitude"] = 0
-                        json_metadata_template["Location.Time"] = 0
-                        heading = row
-
-                    else:
-                        json_metadata_row = copy.deepcopy(json_metadata_template)
-                        column_index = 0
-                        for item in row:
-                            if item.strip() != "":
-                                json_metadata_row[heading[column_index]] = item
-                            column_index += 1
-                        json_metadata["proofs"].append(json_metadata_row)
-
-                json_metadata["pgpSignature"] = pgp
-                json_metadata["pgpPublicKey"] = public_pgp
-                file_hash = json_metadata["proofs"][0]["File Hash SHA256"]
-                json_metadata["sha256hash"] = file_hash
-                json_metadata["dateCreate"] = current_date_create.isoformat()
```

Comment:
This is extracted metadata from proofmode (which we assume may change so its not strict) into higher level keys that we use elseware.